ec2utils/ec2uploadimg/python-ec2uploadimg.spec

```diff
@@ -18,7 +18,7 @@
 
 %define upstream_name ec2uploadimg
 Name:           python-ec2uploadimg
-Version:        2.0.0
+Version:        2.1.0
```

Comment:
Compatible would be if you add running_id at the end of the argument list to EC2ImageUploader, inserting it in the middle makes it an incompatible change. Consider if there is a developer that uses the EC2ImageUploader class and does not use keywork arguments, then what was previously passed as secret_key is now passed as running_id.